tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml.cs

```diff
@@ -0,0 +1,229 @@
+                }
+            }
+        }
+    }
+
+    private async void ExtensionProviderComboBox_Loading(FrameworkElement sender, object args)
+    {
+        // TODO: For a nicer UX, we should enable the user to pick a default provider so that they don't have to select it every time.
+        await ViewModel.PopulateQuickstartProvidersComboBox();
+    }
+
+    private async void ExtensionProviderComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
+    {
+        ViewModel.OnQuickstartSelectionChanged();
+        await ShowExtensionInitializationUI();
+    }
+
+    private void NegativeFeedbackConfirmation_Click(object sender, RoutedEventArgs e)
+    {
+        ViewModel.NegativesGroupOne = false;
+        ViewModel.NegativesGroupTwo = true;
+
+        ViewModel.ProvideFeedback(false, negativeFeedbackTextBox.Text);
+        negativeFeedbackTextBox.Text = string.Empty;
+    }
+
+    private void PositiveFeedbackConfirmation_Click(object sender, RoutedEventArgs e)
+    {
+        ViewModel.ProvideFeedback(true, positiveFeedbackTextBox.Text);
+        positiveFeedbackTextBox.Text = string.Empty;
+
+        ViewModel.PositivesGroupOne = false;
+        ViewModel.PositivesGroupTwo = true;
+    }
+
+    private void NegativeFeedbackFlyout_Closed(object sender, object e)
+    {
+        ViewModel.NegativesGroupOne = true;
+        ViewModel.NegativesGroupTwo = false;
+    }
+
+    private void PositiveFeedbackFlyout_Closed(object sender, object e)
+    {
+        ViewModel.PositivesGroupOne = true;
+        ViewModel.PositivesGroupTwo = false;
+    }
+
+    private void NegCloseFlyout_Click(object sender, RoutedEventArgs e)
+    {
+        negativeFeedbackFlyout.Hide();
+    }
+
+    private void PosCloseFlyout_Click(object sender, RoutedEventArgs e)
+    {
+        positiveFeedbackFlyout.Hide();
+    }
+
+    public void OnAdaptiveCardSessionStopped(IExtensionAdaptiveCardSession2 cardSession, ExtensionAdaptiveCardSessionStoppedEventArgs data)
+    {
+        cardSession.Stopped -= OnAdaptiveCardSessionStopped;
+        if (_adaptiveCardContentDialog is not null)
+        {
+            DispatcherQueue.TryEnqueue(() =>
+            {
+                _adaptiveCardContentDialog?.Hide();
+            });
+        }
+    }
+
+    private async Task ShowAdaptiveCardOnContentDialog(QuickStartProjectAdaptiveCardResult adaptiveCardSessionResult)
+    {
+        if (adaptiveCardSessionResult == null)
+        {
+            // No adaptive card to show (i.e. no dependencies or AI initialization).
+            return;
+        }
+
+        if (adaptiveCardSessionResult.Result.Status == ProviderOperationStatus.Failure)
+        {
+            _log.Error($"Failed to show adaptive card. {adaptiveCardSessionResult.Result.DisplayMessage} - {adaptiveCardSessionResult.Result.DiagnosticText}");
+            return;
+        }
+
+        var adapativeCardController = adaptiveCardSessionResult.AdaptiveCardSession;

```

Comment:
for future: we should probably put a try/catch when we use the adaptive card session since the extension could crash  and this is an out of proc object